src/public/script.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-alert */
```

Comment:
Really not happy I have to give this feedback the 3rd time. Do not disable any eslint rule.